opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1711,8 +1718,41 @@ namespace Opm {
                 //OPM_THROW(std::runtime_error, "Group " + group.name() + "FLD control for injecting groups not implemented" );
             }
 
-        } else if (group.isProductionGroup())
-        {
+            // Handle GCONSALE
+            if (schedule().gConSale(reportStepIdx).has(group.name())) {
```

Comment:
Just to clarify: the code here looks like it has no effect other than throwing if it detects an illegal or unhandled situation. Is that correct?

Reply:
Yes. It is on my todo list.